content/en/blog/_posts/2017-09-00-Introducing-Resource-Management-Working.md

```diff
@@ -3,6 +3,8 @@ title: " Introducing the Resource Management Working Group "
 date: 2017-09-21
 slug: introducing-resource-management-working
 url: /blog/2017/09/Introducing-Resource-Management-Working
+author: >
+   Jeremy Eder (Red Hat)
 ---
 _**Editor's note: today's post is by Jeremy Eder, Senior Principal Software Engineer at Red Hat, on the formation of the Resource Management Working Group**_  
```

Comment:
I'd drop this note. We now just credit the person and optionally their organization.